engine/src/main/java/org/terasology/engine/config/RenderingDebugConfig.java

```diff
@@ -108,6 +108,6 @@ public void setRenderSkeletons(boolean renderSkeletons) {
 
     @Override
     public void propertyChange(PropertyChangeEvent evt) {
-        logger.info("Set {} property to {}. ", evt.getPropertyName().toUpperCase(), evt.getNewValue()); // for debugging purposes
+        logger.info("Set {} property to {}.", evt.getPropertyName().toUpperCase(), evt.getNewValue()); // for debugging purposes
```

Comment:
"for debugging purposes" - should this be a DEBUG log?

Reply:
probably
```suggestion
        logger.debug("Set {} property to {}.", evt.getPropertyName().toUpperCase(), evt.getNewValue());
```